qiskit_experiments/library/randomized_benchmarking/rb_analysis.py

```diff
@@ -87,12 +87,21 @@ def _default_options(cls):
         )
         default_options.plot_raw_data = True
         default_options.result_parameters = ["alpha"]
-        default_options.error_dict = None
-        default_options.epg_1_qubit = None
         default_options.gate_error_ratio = None
+        default_options.gate_counts_per_clifford = None
+        default_options.qpg_1_qubit = None
 
         return default_options
 
+    def set_options(self, **fields):
+        if "error_dict" in fields:
+            warnings.warn(
+                "Option 'error_dict' has been removed and merged into 'gate_error_ratio'.",
```

Comment:
I realized that we don't need metadata dependency. Gate kind information can be extracted from count dict, i.e. counts info in circuit metadata, rather than preparing it on the experiment side with backend.